index.html

```diff
@@ -3,16 +3,16 @@
 <head>
 	<link rel="stylesheet" href="styles.css">
 	<meta charset="UTF-8">
-	<title>Microverse trials</title>
+	<title>Microverse trials review</title>
 </head>
 <body>
 <main id="main">
 	<div id="title-container">
 		<h1 id="title"> Lionel Andres Messi Cuccittini </h1> 
-		<h3> The new king after Maradona </h3>
+		<p> The new king after Maradona </p>
 	</div>
 	<div id= "img-div"> 
-		<img id="image" src="./messi.jpeg" alt=""> 
+		<img id="image" src="./messi.jpeg"> 
```

Comment:
Please add an alt property for every img tag on your document to improve your SEO